src/Input/input.style.js

```diff
@@ -22,9 +23,16 @@ const InputStyles = styled.div`
     font-size: 18px;
     font-weight: normal;
     color: #353f41;
-    padding: 2px 16px;
     transition: border-color 0.2s ease, box-shadow 0.2s ease;
 
+    &.default-padding {
```

Comment:
Also, why not just put this in the parent class and override the style with the icon padding inside that class

Reply:
i dont get that. please elaborate. also do you see something really wrong with the current implementation ?

Reply:
sorry miss-clicked resolve

Reply:
You don’t need the default class it’s redundant. This will also simplify the login in the component.

Reply:
I thought it helps with readability. But but yeah I will go ahead